website/docs/guides/guides-upgrade.md

```diff
@@ -45,77 +45,124 @@ If you executed the previous step already on your server, then obviously nothing
+This is also available at `http(s)://<your-domain>/index.php?mod=install&action=db`.
 
 ### Test your installation
 
 Now is the time to look if everything is as expected.
 If yes, then you are good to unlock the installation.
 If not then either see what broke in the various logs or go to Rollback.
 
-## Rollback (if stuff doesn't work)
+### Unlock installation
+
+If you locked your installation, you  can now unlock it again.
+"Admin-Page" -> "Common Settings" -> "Lock LanSuite page".
+
+## Rollback (if things doesn't work)
 
-In case your upgrade failed terribly, and you need to return to your old state the following needs to be done:
+In case your upgrade failed, and you need to return to your old state the following needs to be done:
 
 ### Import DB backup
 
-You have to reimport the database backup taken before. How this is done varies in what tools you have available.
-The command line call for this would be `mysql -u<ls_user> -p < exportfile.sql`
+You have to reimport the database backup taken before.
+How this is done varies in what tools you have available.
+The command line call for this would be `mysql -u<ls_user> -p < exportfile.sql`.
 
 ### Restore folder backup
 
 Next step is to restore the folder backup taken.
-First clean up (`rm -rf <failed_update_folder>`) or move away (` mv <failed_update_folder> <anyothername>`) the failed installation.
+First clean up (`rm -rf <failed_update_folder>`) or move away (`mv <failed_update_folder> <anyothername>`) the failed installation.
 Then restore the original installation from your backup.
 Either by extracting (`gunzip ls_backup.tar.gz && tar xvf ls_backup.tar`) or moving back the copied folder (`mv <ls_folder>.backup <ls_folder>`).
 Or: Re-uploading your backup, if that was the way you went.
 
-### Unlock installation
+## Pitfalls and Known Bugs
 
-As the backup was taken at a point where the installation was locked, it may still be required to unlock it
-This would be done by reverting whatever was done to facilitate the original lock
+* Please ensure that export and import of Database images use the same character encoding. Using the same client on the same system should ensure this, but be cautious if dump and import are done on different systems/clients.
+* The git `master` branch is not usable without pulling in additional resources via composer. You must do this first to obtain a runnable installation!
 
-## Pitfalls and Known Bugs
+## Upgrade from LanSuite v4.2 to v5.0
+
+### Configuration: `database.sqlmode`
+
+Add the following line to your configuration at `/inc/base/config.php`:
+
+```
+[...]
+[database]
+sqlmode = "NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION"
+[...]
+```
+
+### Configuration: `database.charset`
+
+Add the following line to your configuration at `/inc/base/config.php`:
+
+```
+[...]
+[database]
+charset = "utf8mb4"
```

Comment:
There was a discussion in the past about what the default value should be.
`utf8mb4` is set by default, but if I'm not mistaken that causes display issues on old installations where content is stored as `latin1`. Unless I missed that being fixed in the meantime

Reply:
This is true - Mainly to a different charset.
I do think we should not take responsibility on converting a database for others. I think there are better articles in the internet that cover this.

Maybe adding one sentence to the doc to mark this?

Reply:
I think we discussed somewhere that we intend to cater also to not that tech-savvy users.
Thus it would be hard to expect them to figure that out for themselves.

Interestingly there were already some on the old page
https://web.archive.org/web/20090113122545/http://lansuite.orgapage.de/index.php?mod=wiki&action=show&name=Latin1toutf8
This we would just need to translate and integrate.
But yes, there should be at least a remark and the recommendation to go with `latin1` if issues occurr and/or unable to convert

Reply:
I've added a bit of text in that regard. 
Feel free to revert, modify or accept

Reply:
> I think we discussed somewhere that we intend to cater also to not that tech-savvy users.

That is a good requirement, however, I don't see this realistic in the current (and near future) state of this software. There is tons of work to do to make this happen. I won't be able to invest this amount of time.

@M4LuZ I see you did a change already: https://github.com/lansuite/lansuite/pull/794/commits/df478fba540978eeed34a02d713230ddb65be7d0

I think it is the wrong direction, however, if you like to have it this way, I am fine with it.

Reply:
> There is tons of work to do to make this happen. I won't be able to invest this amount of time.

Neither will I. But in this case it was just a few minutes to add the related text.

> if you like to have it this way, I am fine with it.

So am I ;)

> Feel free to revert, modify or accept